spec/factories/intercept_message_factory.rb

```diff
@@ -33,6 +33,12 @@
       end
     end
 
+    trait :with_section_to_transform do
```

Comment:
I'd hoist this above chapter to reflect the hierarchical position of the section but that's a small preference/semantic thing